engine/classes/ElggInstaller.php

```diff
@@ -356,6 +356,12 @@ protected function database($submissionVars) {
 				'value' => 'elgg_',
 				'required' => TRUE,
 				),
+			'timezone' => array(
+				'type' => 'dropdown',
+				'value' => 'UTC',
+				'options' => \DateTimeZone::listIdentifiers(),
+				'required' => TRUE
+			)
```

Comment:
What about cases where two systems located in different timezones communicate with each other?